islandora_memberofsite.module

```diff
@@ -49,33 +49,24 @@ function islandora_memberofsite_menu() {
  */
 function islandora_memberofsite_islandora_overview_object_alter(
   AbstractObject &$object, &$output) {
-  // To map the isMemberOfSite resource value to the underlying public-facing
-  // islandora site.
-  $site_mappings = array(
-    'pitt:site.uls-digital-collections' => 'Digital Collections',
-    'pitt:site.documenting-pitt' => 'Documenting Pitt',
-    'pitt:site.historic-pittsburgh' => 'Historic Pittsburgh',
-    'pitt:site.american-music' => 'American Music',
-  );
   module_load_include('inc', 'islandora_memberofsite', 'includes/utilities');
   if (!user_access(ISLANDORA_MEMBEROFSITE_MANAGE)) {
     return;
   }
-  $obj_sites = $object->relationships->get(variable_get('islandora_memberofsite_namespace', NULL), NULL);
+  $sites = islandora_memberofsite_get_sites();    
+  $obj_sites = $object->relationships->get(NULL, 'isMemberOfSite');
```

Comment:
Why pass `NULL` as a namespace here rather than use `variable_get('islandora_memberofsite_namespace')`?

The use of the string "isMemberOfSite" suggests the need for a code constant or a Drupal variable, but fixing this is deferrable.